filters/filter_blur.py

```diff
@@ -0,0 +1,144 @@
+import cv2
+import numpy
+
+from filters.filter import Filter
+
+
+class FilterBlur(Filter):
+    """
+    https://medium.com/@stepanfilonov/tracking-your-eyes-with-python-3952e66194a6
+    """
+
+    face_cascade = cv2.CascadeClassifier('img/haarcascade_frontalface_default.xml')
+
+    def __init__(self):
+        super(FilterBlur, self).__init__(duration=0)
+        self.do_stop = False
+        self.debug = False
+
+    def get_image(self, frame, faces):
```

Comment:
renamed "pixelize" to "get_image" and made it not static so that I could override it in filter_blur_background (and have the naming make sense). It's the only method with different logic.